flume-ng-sinks/flume-http-sink/pom.xml

```diff
@@ -30,11 +30,6 @@ limitations under the License.
   <artifactId>flume-http-sink</artifactId>
   <name>Flume HTTP/S Sink</name>
 
-  <properties>
-    <wiremock.version>1.53</wiremock.version>
-    <guava.version>18.0</guava.version>
-  </properties>
-
```

Comment:
Now, that we use common versions, we might would like to look at the exlusions in the dependencies. Again, the version numbers could come from `dependencyManagement`